partials/shop-invoicetotals.htm

```diff
@@ -5,22 +5,32 @@
     {% set cartTotalNoTax = cartTotalNoTax + item.price %}
   {% endfor %}
 {% endfor %}
-
+{% set discount = totals.discount_total %}
 {% if cartTotalNoTax > 0 %}
-    <li class="list-group-item"><strong>Cart Total:</strong> <i>{{ cartTotalNoTax|currency }}</i></li>
+    <li class="list-group-item">Item Total: <div class="pull-right">{{ cartTotalNoTax|currency }}</div></li>
```

Comment:
Should this be "Item Total:"? Seems like `{{ cartTotalNoTax|currency }}` should be the total of the order, as opposed to the item.